WordPress/Classes/ViewRelated/Jetpack/Jetpack Restore/Restore Status/RestoreStatusView.swift

```diff
@@ -0,0 +1,80 @@
+import Foundation
+import Gridicons
+import WordPressUI
+
+class RestoreStatusView: UIView, NibLoadable {
+
+    // MARK: - Properties
+
+    @IBOutlet private weak var icon: UIImageView!
+    @IBOutlet private weak var title: UILabel!
+    @IBOutlet private weak var body: UILabel!
+    @IBOutlet private weak var progressTitle: UILabel!
+    @IBOutlet private weak var progressValue: UILabel!
+    @IBOutlet private weak var progressView: UIProgressView!
+    @IBOutlet private weak var progressDescription: UILabel!
+    @IBOutlet private weak var notifyMeButton: UIButton!
+    @IBOutlet private weak var hint: UILabel!
+
+    var notifyMeHandler: (() -> Void)?
+
+    // MARK: - Initialization
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        applyStyles()
+        configure()
+    }
+
+    // MARK: - Styling
+
+    private func applyStyles() {
+        backgroundColor = .white
```

Comment:
For light/dark mode:

```suggestion
        backgroundColor = .basicBackground
```